schemas/list-workertypes-response.yml

```diff
@@ -20,10 +20,45 @@ properties:
           minLength:    {$const: identifier-min-length}
           maxLength:    {$const: identifier-max-length}
           pattern:      {$const: identifier-pattern}
+        provisionerId:
+          title:        "Provisioner ID"
+          type:         string
+          minLength:    {$const: identifier-min-length}
+          maxLength:    {$const: identifier-max-length}
+          pattern:      {$const: identifier-pattern}
+        stability:
+          title:        "Stability"
+          description: |
+            The stability of the worker-type. Accepted values:
+            'experimental|stable|deprecated'
+          type:         string
+          enum:         ["experimental", "stable", "deprecated"]
+        description:
+          title:        "Description"
+          description: |
+            Description of the worker-type.
+          type:         string
+        documentation:
```

Comment:
@djmitche What do you mean with "same REST resource, just different views". Did you mean the following:
`GET /v1/listWorkerTypes` omits documentation and payload
`GET /v1/provisioners/:provisionerId/worker-types/:workerType` includes documentation and payload

If that's not what you meant, would ^ be a viable option? In that case, `GET /v1/provisioners/:provisionerId/worker-types/` would perhaps also omit documentation and payload.